documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -40,6 +42,79 @@ rpi-eeprom-digest -i pieeprom.upd -o pieeprom.sig
 rpi-eeprom-digest -i boot.img -o boot.sig -k myprivkey.pem
 ----
 
+=== Certificates
+
+For security, Network Install uses HTTPS to download OS images from the Raspberry Pi website. This feature uses our own CA root included in the bootloader to verify the host.
+
+You can add your own custom CA certificate to the EEPROM to securely download images from your own website. Use the `--cacertder` option of the `rpi-eeprom-config` tool to add the DER-encoded certificate.
+
+To ensure that the certificate is not modified, place a hash of the certificate in the EEPROM config settings.
+
+Run the following command to generate a DER-encoded certificate:
+
+[source,console]
+----
+$ openssl x509 -in your_ca_root_cert.pem -out cert.der -outform DER
+----
+
+Then, run the following command to generate a SHA-256 hash of the certificate:
+
+[source,console]
+----
+$ sha256sum cert.der
+----
+
+You should see output similar to the following:
+
+----
+701bd97f67b0f5483a9734e6e5cf72f9a123407b346088638f597878563193fc  cert.der
+----
+
+Next, update `boot.conf` to include the hash of the certificate:
+
+[source,console]
+----
+$ sudo rpi-eeprom-config --edit
+----
+
+Configure the following settings in the `[gpio8=0]` section, replacing:
+
+* `<your_website>` with your website, e.g. `raspberrypi.com`
+* `<path_to_image>` with the path to your OS image hosted on your website, e.g. `path/to/image`
+* `<hash>` with the hash value you generated above, e.g. `701bd97f67b0f5483a9734e6e5cf72f9a123407b346088638f597878563193fc`
+
+----
+[all]
+BOOT_UART=1
+POWER_OFF_ON_HALT=0
+BOOT_ORDER=0xf461
+
+[gpio8=0]
+BOOT_ORDER=0xf7
+NET_INSTALL_ENABLED=0
+HTTP_HOST=<your_website>
+HTTP_PATH=<path_to_image>
+HTTP_CACERT_HASH=<hash>
+----
+
+NOTE: If you specify a `HTTP_CACERT_HASH`, https downloads are enabled and port 443 is used instead of port 80.
```

Comment:
```suggestion
When you specify a `HTTP_CACERT_HASH`, Network Install downloads the image using HTTPS over port 443 instead of HTTP over port 80.
```